target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_arcadyan_vrv9510kwac23.dtsi

```diff
@@ -0,0 +1,284 @@
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+
+		rfkill {
+			label = "rfkill";
+			gpios = <&gpio 5 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RFKILL>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		// from top to bottom
+		power_green: power {
+			label = "green:power";
+			gpios = <&gpio 38 GPIO_ACTIVE_LOW>;
+			default-state = "keep";
+		};
+		internet_green: internet_green {
+			label = "green:internet";
+			gpios = <&gpio 26 GPIO_ACTIVE_LOW>;
+		};
+		internet_red {
+			label = "red:internet";
+			gpios = <&gpio 33 GPIO_ACTIVE_LOW>;
+		};
+		phone {
+			label = "green:phone";
+			gpios = <&gpio 44 GPIO_ACTIVE_LOW>;
+		};
+		lan {
+			label = "green:lan";
+			gpios = <&gpio 45 GPIO_ACTIVE_LOW>;
+		};
+		wireless_blue {
+			label = "blue:wlan";
+			gpios = <&gpio 40 GPIO_ACTIVE_LOW>;
+		};
+		wireless_green: wireless_green {
+			label = "green:wlan";
+			gpios = <&gpio 47 GPIO_ACTIVE_LOW>;
+		};
+		upgrade: upgrade {
+			label = "blue:upgrade";
+			gpios = <&gpio 41 GPIO_ACTIVE_LOW>;
+		};
+		
+	};
+	
+	bcm43222-sprom {
+		compatible = "brcm,bcm43222-sprom";
+
+		pci-bus = <0>;
+		pci-dev = <1>;
+
+		//nvmem-cells = <&macaddr_cfe_6a0>;
+		//nvmem-cell-names = "mac-address";
+		//mac-address-increment = <1>;
+		brcm,sprom-fixups = <97 0xfeb3>,
+				    <98 0x1618>,
+				    <99 0xfab0>,
+				    <113 0xfed1>,
+				    <114 0x1609>,
+				    <115 0xfad9>;
+	
+	};
+
+	usb_vbus: regulator-usb-vbus {
+		compatible = "regulator-fixed";
+
+		regulator-name = "USB_VBUS";
+
+		regulator-min-microvolt = <5000000>;
+		regulator-max-microvolt = <5000000>;
+
+		gpio = <&gpio 32 GPIO_ACTIVE_HIGH>;
+		enable-active-high;
+	};
+	
+	
+};
+
+&localbus {
+	flash@0 {
+		compatible = "lantiq,nand-xway";
+		lantiq,cs = <1>;
+		bank-width = <2>;
+		reg = <0 0x0 0x2000000>;
+
+		pinctrl-0 = <&nand_pins>, <&nand_cs1_pins>;
+		pinctrl-names = "default";
+
+		nand-use-soft-ecc-engine;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "uboot";
+				reg = <0x00000 0x80000>;
+			};
+			partition@40000 {
+				label = "u-boot-env";
+				reg = <0x80000 0x20000>;
+			};
+			partition@60000 {
+				label = "kernel";
+				reg = <0xA0000 0x3A0000>;
+			};
+			partition@360000 {
+				label = "ubi";
+				reg = <0x3A0000 0x7c60000>;
+			};
+		};
+	};
+};
+
+
+
+&gphy0 {
+	lantiq,gphy-mode = <GPHY_MODE_GE>;
+};
+
+&gphy1 {
+	lantiq,gphy-mode = <GPHY_MODE_GE>;
+};
+
+&gpio {
+	pinctrl-names = "default";
+	pinctrl-0 = <&state_default>;
+
+	state_default: pinmux {
+		exin3 {
+			lantiq,groups = "exin3";
+			lantiq,function = "exin";
+		};
+		pci_rst {
+			lantiq,pins = "io21";
+			lantiq,output = <1>;
+			lantiq,open-drain = <0>;
+			lantiq,pull = <2>;
+		};
+		pcie-rst {
+			lantiq,pins = "io38";
+			lantiq,pull = <0>;
+			lantiq,output = <1>;
+		};
+	};
+};
+
+&gswip {
+	pinctrl-0 = <&mdio_pins>;
+	pinctrl-names = "default";
+};
+
+&gswip_mdio {
+	phy0: ethernet-phy@0 {
+		reg = <0x0>;
```

Comment:
This PHY may have delays set up via boot strap lines. Check on kernel 5.15 if similar logs are displayed during boot:
```
[    2.269736] Intel XWAY PHY11G (PEF 7071/PEF 7072) v1.5 / v1.6 1e108000.switch-mii:00:
               PHY has delays (e.g. via pin strapping), but phy-mode = 'rgmii'
[    2.269736] Should be 'rgmii-id' to use internal delays txskew:1500 ps rxskew:1500 ps
```
Example: https://git.openwrt.org/?p=openwrt/openwrt.git;a=commit;h=069bda0adb5cb97c78ad9ea2561512ced487fa59

Reply:
Done